### ISSUE TITLE
Ansible dry-run failed on pkg-redhat.yml

### ISSUE THREAD

**Issue by chaosoldier** [DataDog/ansible-datadog#75]
"Import new RPM key" task was failed when I run "tasks/pkg-redhat.yml" with dry-run mode. Because dry-run mode had not made the fie "/tmp/DATADOG_RPM_KEY_E09422B3.public". 
And, sometimes any files in /tmp directory may be deleted by setting. 
So, I add  "always_run: yes" to get_url task . (in 2.X must change to check_mode: no) 

```
- name: Download new RPM key
  get_url:
    url: "http://yum.datadoghq.com/DATADOG_RPM_KEY_E09422B3.public"
    dest: /tmp/DATADOG_RPM_KEY_E09422B3.public
    sha256sum: 694a2ffecff85326cc08e5f1a619937999a5913171e42f166e13ec802c812085

- name: Import new RPM key
  rpm_key: key=/tmp/DATADOG_RPM_KEY_E09422B3.public state=present
```

**Comment by olivielpeau** [comment]
Hi @chaosoldier, thanks for reporting this!

Feel free to open a PR fixing this since you seem to have a good idea of how to do it.

Otherwise I'll work on a fix in the future ;)

**Comment by hush-hush** [comment]
Hi @chaosoldier,

PR #128 should fix this issue. Feel free to comment on it !

**Comment by hush-hush** [comment]
#128 was merged and release with version 2.1.0 ansible role.

Thanks for opening this issue !
